### DISCUSSION TITLE
AngularFire docs in Spanish.

### DISCUSSION THREAD

**Discussion by Villanuevand** [angular/angularfire#3239] [General]
Hello peeps! 

2 years ago I started to work on translating the documentation of this repo to [Spanish](https://github.com/angular/angularfire/pull/2547#issue-660449972). 

I would like to know if anyone is interested to help in this adventure.  So far I identified 23 docs to translate. 

- [x]  [Quickstart](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/install-and-setup.md) Commit: e12c835
- [x]  [Upgrading to v7.0? Check out our guide](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/version-7-upgrade.md) Commit: 6aa203f
- [x]  [Getting started with Google Analytics](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/analytics/getting-started.md) Commit:  d6abd13
- [ ]  [Documents](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/firestore/documents.md)
- [ ]  [Collections](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/firestore/collections.md)
- [ ]  [Querying Collections](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/firestore/querying-collections.md)
- [ ]  [Offline data](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/firestore/offline-data.md)
- [ ]  [Objects](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/rtdb/objects.md)
- [ ]  [Lists](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/rtdb/lists.md)
- [ ]  [Querying lists](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/rtdb/querying-lists.md)
- [ ]  [Getting started with Firebase Authentication](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/auth/getting-started.md)
- [ ]  [Route users with AngularFire guards](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/auth/router-guards.md)
- [ ]  [Getting started with Firebase Emulator Suite](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/emulators/emulators.md)
- [ ]  [Getting started with Cloud Storage](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/storage/storage.md)
- [ ]  [Getting started with Firebase Messaging](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/messaging/messaging.md)
- [ ]  [Getting started with Callable Functions](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/functions/functions.md)
- [ ]  [Deploy your application on Firebase Hosting](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/deploy/getting-started.md)
- [ ]  [Getting started with Angular Universal](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/universal/getting-started.md)
- [ ]  [Deploying your Universal application on Cloud Functions for Firebase](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/universal/cloud-functions.md)
- [ ]  [Prerendering your Universal application](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/universal/prerendering.md)
- [ ]  [Installation and Setup with Ionic CLI](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/ionic/cli.md)
- [ ]  [Using AngularFire with Ionic 2](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/ionic/v2.md)
- [ ]  [Using AngularFire with Ionic 3](https://github.com/angular/angularfire/blob/a6abd1d961d18b5c32d6f56fb2f47b251934d6af/docs/ionic/v3.md)

This work will help more dev to use Angular + Firebase properly without a language barrier.
If you want to contribute just reply this thread and let's talk about it. 

Cheers!

**Comment by ernestocardenas79** [comment]
que onda pana !!! cual es la mecanica o que show??

**Reply by Villanuevand** [reply]
Hola Ernesto @ernestocardenas79 , 
Vamos a usar ingles acá por ser el canal oficial. 

@ernestocardenas79  The main idea of this initiative is translate the entire documentation in this repo to Spanish. 
If you're interesting to contribute let me know, we can schedule a meet to talk about it.

**Reply by ernestocardenas79** [reply]
great, do you have any video where can i get the basic process??

**Reply by Villanuevand** [reply]
I don't have a video for it but I can explain it! 
### To contribute please: 
- Fork https://github.com/Villanuevand/angularfire
- The `main` branch for this issue is: [feat-readme-es](https://github.com/Villanuevand/angularfire/tree/feat-readme-es)
- Create a new branch base on `feat-readme-es` and add your work.
- Push your changes. 

After check the changes I'll push to this repo your changes. 

If something is not clear, Let me now to schedule a call.

**Comment by jlbousing** [comment]
This is my first contribution with angular. This gonna be great. Can I start translate document or collection as my first contribution ?

**Reply by Villanuevand** [reply]
@jlbousing yes sure, 
Welcome to the Open source contributors world... :) 

To contribute please: 
- Fork https://github.com/Villanuevand/angularfire
- The `main` branch for this issue is: [feat-readme-es](https://github.com/Villanuevand/angularfire/tree/feat-readme-es)
- Create a new branch base on `feat-readme-es` and add your work.
- Push your changes. 

After check the changes I'll push to this repo your changes. 

If something is not clear, Let me now to schedule a call.
